src/_tutorials/web/low-level-html/add-elements.md

```diff
@@ -432,7 +432,7 @@ generateNewLetters() {
 {% endcomment %}
 
 <iframe
-src="{{site.custom.dartpad.embed-html-prefix}}?id=0532bfcb70bf5e4a900c&horizontalRatio=70&verticalRatio=80"
+src="{{site.custom.dartpadx.embed-html-prefix}}?id=0532bfcb70bf5e4a900c"
```

Comment:
The console isn't useful here.

Reply:
We can't hide the console yet, but I've added that to the list: https://github.com/dart-lang/dart-pad/issues/1185